modules/backend/classes/NavigationManager.php

```diff
@@ -16,15 +18,15 @@
  */
 class NavigationManager
 {
-    use \October\Rain\Support\Traits\Singleton;
```

Comment:
It's October's style to use the full namespace for model traits instead of importing it at the top, so please revert this.

Reply:
I've reverted this in a new commit, guess this is something my IDE does by itself.